app/helpers/trends_helper.rb

```diff
@@ -33,8 +33,15 @@ def chart_data(trend, from = Setting.max_trend, to = Time.zone.now)
     values = trend.values
     labels = {}
     values.includes(:trendable).each {|v| labels[v.id] = [v.to_label, trend_path(:id => v)]}
-    values.includes(:trend_counters).where(["trend_counters.created_at > ?", from]).reorder("trend_counters.created_at").each_with_index.map do |value, idx|
-      data =  value.trend_counters.map { |t|  [t.created_at.to_i*1000, t.count]  }
+    values.includes(:trend_counters).where(["trend_counters.interval_start > ? or trend_counters.interval_end > ? or trend_counters.interval_end is null", from, from]).reorder("trend_counters.interval_start").each_with_index.map do |value, idx|
```

Comment:
is in on purpose that both interval_start and interval_end are bigger then "from" shouldnt it be one bigger and the other smaller then?